### DISCUSSION TITLE
Base64 encode/decode Twig filters

### DISCUSSION THREAD

**Discussion by elivz** [craftcms/cms#12696] [Ideas] [answered]
Would be handy (and very easy) to have `base64_encode` and `base64_decode` Twig filters/functions built in. I've added this a few times in a custom site-functionality module, and it's a one-liner. But would be even better not to have to!

```
new TwigFilter("base64_encode", "base64_encode"),
new TwigFilter("base64_decode", "base64_decode"),
```

**Comment by brandonkelly** [comment, marked as answer]
Added for Craft 4.4! (Via 773c1a438bba9e03c28effdd27ba58bdf1dbb97b)

**Reply by elivz** [reply]
Brandon, you're a legend!

**Reply by brandonkelly** [reply]
4.4 is out!
